socrata-http-client/src/main/scala/com/socrata/http/client/HttpClientHttpClient.scala

```diff
@@ -33,12 +35,26 @@ class HttpClientHttpClient(executor: Executor, options: HttpClientHttpClient.Opt
   import HttpClient._
   import options._
 
+  private[this] val sslFactory = new SSLConnectionSocketFactory(
+    sslContext,
+    Array("TLSv1", "TLSv1.1", "TLSv1.2"),
+    null,
```

Comment:
The original SSLConnectionSocketFactory w/o arguments pass null to the one with this argument.  It seems to mean do not override the socket cipher whatever it is.